avm/res/network/private-endpoint/main.bicep

```diff
@@ -9,37 +9,39 @@ param name string
 param subnetResourceId string
 
 @description('Optional. Application security groups in which the private endpoint IP configuration is included.')
-param applicationSecurityGroupResourceIds array?
+param applicationSecurityGroupResourceIds string[]?
 
 @description('Optional. The custom name of the network interface attached to the private endpoint.')
 param customNetworkInterfaceName string?
 
 @description('Optional. A list of IP configurations of the private endpoint. This will be used to map to the First Party Service endpoints.')
-param ipConfigurations ipConfigurationsType
+param ipConfigurations ipConfigurationType[]?
 
 @description('Optional. The private DNS zone group to configure for the private endpoint.')
 param privateDnsZoneGroup privateDnsZoneGroupType?
 
 @description('Optional. Location for all Resources.')
 param location string = resourceGroup().location
 
+import { lockType } from 'br/public:avm/utl/types/avm-common-types:0.1.0'
```

Comment:
For easier maintenance, would it make sense to have only one import line with multiple types?

Reply:
Yes and no. So far I'm torn on this one. The benefit of having all in one is, like you said, to well - have it all in one. But the question then is where to put it. 
- Anywhere? I guess that would negate the maintanence benefit.
- Then maybe in the `// Definitions //` section. But what if a module doesn't have any other definitions. Should we then add such as section? And if we do, does that mean we have to define a spec that all type imports always have to be in a definitions section...

For now, I opted to put it right next to the parameter where it is used. Aside from the strong coupling, it has the one benefit that you can have the option to vary the version you want to use per type, instead of having to use one module version for all types.

Reply:
@ReneHezser @AlexanderSehr Good question and discussion. I'd also be more inclined towards having one import for each parameter using it. I'd actually see it more convenient from a maintenance perspective as well.